Example-iOS/Podfile

```diff
@@ -0,0 +1,9 @@
+source 'https://github.com/CocoaPods/Specs.git'
+
+target 'VOKBenkode', :exclusive => true do
```

Comment:
This seems to bump into a thing I've encountered before—if the `Podfile` doesn't mention the test target at all, the test target doesn't get properly configured to work with pods (I think it doesn't get set up with the configuration to have the header search paths, etc.).